intranet/static/js/common.nav.js

```diff
@@ -12,21 +12,22 @@ $(function() {
 
     $(".main *:horizontallyscrollable").on("touchstart", function(e) {
         var scrollPos = $(this).scrollLeft();
-        if (scrollPos != 0) {
+        if (scrollPos !== 0) {
             e.stopPropagation();
         }
     });
 
     var initX = null, initY = null, listening = false;
-    $(window).on("touchstart", function(e) {
+    var win = $(window);
+    win.on("touchstart", function(e) {
         initX = e.originalEvent.touches[0].clientX;
         initY = e.originalEvent.touches[0].clientY;
         listening = true;
     });
-    $(window).on("touchend", function(e) {
+    win.on("touchend", function() {
         listening = false;
     });
-    $(window).on("touchmove", function(e) {
+    win.on("touchmove", function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 6.](https://www.codacy.com/app/tjhsst-ion/ion/file/2632561822/issues/source?bid=3212528&fileBranchId=3299315#l30)